cirq-core/cirq/devices/line_qubit_test.py

```diff
@@ -287,18 +287,27 @@ def test_numeric():
     assert isinstance(complex(cirq.LineQubit(x=5)), complex)
 
 
-def test_numpy_index():
-    np5 = np.int64(5)
+@pytest.mark.parametrize('dtype', (np.int8, np.int64, float, np.float64))
+def test_numpy_index(dtype):
+    np5 = dtype(5)
     q = cirq.LineQubit(np5)
     assert hash(q) == 5
     assert q.x == 5
+    assert q.x == dtype(5)
```

Comment:
Do we need this after the previous line?  NumPy gives True for
`np.int8(5) == np.int64(5) == float(5) == np.float64(5) == 5`

Reply:
Yeah, not really. Removed.